app/helpers/application_helper/button/generic_object_definition_button_button_group_delete.rb

```diff
@@ -1,5 +1,23 @@
 class ApplicationHelper::Button::GenericObjectDefinitionButtonButtonGroupDelete < ApplicationHelper::Button::Basic
+  needs :@record
+
   def disabled?
-    !@record.custom_buttons.count.zero?
+    @record.kind_of?(CustomButtonSet) && !@record.custom_buttons.count.zero?
```

Comment:
Is this right? Should the button really be enabled whenever `@record` is **not** a `CustomButtonSet`?

It seems like this change will make the button available when @record is a CustomButton,
but that's probably wrong since the toolbar definition talks about the button being only for button groups.

Reply:
Still, this seems like it would be easier to have 2 different buttons, one for CustomButtonSet, with the disable logic,
and the other for CustomButton.

Then, you could actually provide the translated strings instead of relying on `record.class.model_name.plural` or `record.class.name`.

Reply:
Resolved